types/youtube-player/dist/types.d.ts

```diff
@@ -42,6 +42,11 @@ export interface IframeApiType {
     Player: {new(elementId: string, options: Options): YouTubePlayer};
 }
 
+export interface sizeType {
```

Comment:
Please use naming conventions in line of other identifiers, and prefer context-conveying names. How about `PlayerSize`?

Reply:
careless camelCase...amended!